quadriga/src/main/java/edu/asu/diging/quadriga/api/v1/AddNetworkApiController.java

```diff
@@ -53,11 +79,15 @@ public HttpStatus processJson(@RequestBody Quadruple quadruple) {
         // save network
         List<CreationEvent> events = networkMapper.mapNetworkToEvents(quadruple.getGraph());
         List<EventGraph> eventGraphs = events.stream().map(e -> new EventGraph(e)).collect(Collectors.toList());
-        eventGraphs.forEach(e -> e.setDefaultMapping(quadruple.getGraph().getMetadata().getDefaultMapping()));
+        eventGraphs.forEach(e -> {
+            e.setCollectionId(mappedTripleGroup.getCollectionId());
```

Comment:
why don't you use `collectionId` here? This makes it look like `collectionId` is different than `mappedTripleGroup.getCollectionId()`.